.github/workflows/build-lint-test.yml

```diff
@@ -180,7 +176,16 @@ jobs:
           node-version-file: '.nvmrc'
           cache: yarn
       - run: yarn --immutable --immutable-cache
-      - run: yarn workspace ${{ matrix.package-name }} run lint:ci
+      - name: ESLint
```

Comment:
Why not just run `yarn lint` directly?

Reply:
I was thinking it could be nice to see which part is failing when looking at the workflow, but happy to change it back if you prefer.

Reply:
Yeah, the only problem is that we have to remember to update CI too if we modify the linting commands